pages/accept-financial-contributions.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from '@apollo/react-hoc';
+
+import AcceptFinancialContributions from '../components/accept-financial-contributions/index.js';
+import ErrorPage from '../components/ErrorPage';
+import AuthenticatedPage from '../components/AuthenticatedPage';
+import { withUser } from '../components/UserProvider';
+import Loading from '../components/Loading';
+import Container from '../components/Container';
+
+import { getCollectivePageQuery } from '../components/collective-page/graphql/queries';
+
+class AcceptFinancialContributionsPage extends React.Component {
+  static async getInitialProps({ query }) {
+    return {
+      slug: query && query.slug,
+      query,
+    };
+  }
+
+  static propTypes = {
+    query: PropTypes.object,
+    slug: PropTypes.string, // for addCollectiveCoverData
+    data: PropTypes.object, // from withData
+    loadingLoggedInUser: PropTypes.bool,
+    LoggedInUser: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { data, query } = this.props;
+
+    if (!data.loading) {
+      if (!data || data.error) {
+        return <ErrorPage data={data} />;
+      }
+    }
+
+    const collective = data && data.Collective;
+    return (
+      <AuthenticatedPage>
+        {data.loading ? (
+          <Container py={[5, 6]}>
+            <Loading />
+          </Container>
+        ) : (
+          <AcceptFinancialContributions query={query} collective={collective} />
+        )}
+      </AuthenticatedPage>
+    );
+  }
+}
+
+const getCollective = graphql(getCollectivePageQuery, {
+  options: props => ({
```

Comment:
Can you try without the `options`, I think the props are automatically used as variables.

Reply:
Sorry could you explain what you mean? Looking at the codebase most graphql queries have the props passed in via options.

Reply:
I only found that in an old documentation but it's certainly still true today.

> By default, graphql will attempt to pick up any missing variables from the query from ownProps. [...] However, if you need to change the name of a variable, compute the value, or just want to be more explicit about things, the options function is the place to do it.

We're relying on that in many places through the codebase, I find it elegant.